.build/release.props

```diff
@@ -8,7 +8,7 @@
     <Product>DarkLoop's Azure Functions Authorization</Product>
     <IsPreview>false</IsPreview>
```

Comment:
can we flip this flag to true first? to produce a preview package and ensure everything is good.
After that I will flip it to false to get the release version

Reply:
Will do!